weave

```diff
@@ -797,6 +797,25 @@ add_iface_bridged_fastdp() {
     add_iface_bridge "$@"
 }
 
+attach_bridge() {
+    bridge="$1"
+    LOCAL_IFNAME=v${CONTAINER_IFNAME}bl$bridge
+    GUEST_IFNAME=v${CONTAINER_IFNAME}bg$bridge
+
+    ip link del $LOCAL_IFNAME >/dev/null 2>&1 || true
+    ip link del $GUEST_IFNAME >/dev/null 2>&1 || true
+
+    ip link add name $LOCAL_IFNAME mtu $MTU type veth peer name $GUEST_IFNAME mtu $MTU || return 1
+
+    if ! ip link set $LOCAL_IFNAME up ||
+       ! ip link set $GUEST_IFNAME up ||
+       ! add_iface_$BRIDGE_TYPE $LOCAL_IFNAME ||
+       ! ip link set $GUEST_IFNAME master $bridge ; then
+        ip link del $GUEST_IFNAME >/dev/null 2>&1 || true
+        return 1
+    fi
+}
```

Comment:
That code looks like it does the same as `setup_router_iface_bridge`, modulo veth pair naming.